jb/fftw/ut_tde_result.cpp

```diff
@@ -0,0 +1,111 @@
+#include <jb/detail/array_traits.hpp>
+#include <jb/fftw/aligned_multi_array.hpp>
+#include <jb/fftw/tde_result.hpp>
+
+#include <boost/test/unit_test.hpp>
+
+/**
+ * @test Verify that we can create a 2 dim tde_result
+ */
+BOOST_AUTO_TEST_CASE(fftw_tde_result_2_dim_usage) {
+  using array_type = jb::fftw::aligned_multi_array<float, 3>;
+  using test_tde_result = jb::fftw::aligned_multi_array<std::size_t, 2>;
+  using tde_result_type = jb::fftw::tde_result<array_type, std::size_t>;
+  using tde_result_type = jb::fftw::tde_result<array_type, std::size_t>;
+
+  // create a tde_result based on array_type
+  bool res = std::is_same<tde_result_type::record_type, test_tde_result>::value;
+  BOOST_CHECK_MESSAGE(
+      res == true, "tde_result<array_type> is not multi array float");
+
+  // create a test timeseries
+  const int O = 10;
```

Comment:
Please consider making the dimensions different, e.g., 5, 10, 15.  In case we have code that confuses one dimension with another, having all the same would miss such a mistake.